### PR TITLE
Remove `encryption` and `auth_mode` as they are removed since Symfony 4.4.0

### PR DESCRIPTION
**Pull request by simonschaufi**
See https://github.com/symfony/mailer/blob/6.3/CHANGELOG.md#440

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thanks. Even better would be a link to the docs. encryption=tls was replaced by smtps:// if I remember correct?!

**Comment by simonschaufi** [comment]
Added the source.

> encryption=tls was replaced by smtps:// if I remember correct?!

yes

File: _documentation/emails.md [hunk 1]
```diff
@@ -83,6 +83,6 @@ app.ERROR: Exception occurred while flushing email queue: Expected response code
 ```
 you might suffer from a wrong configuration (read the documentation linked above) and try a fully featured SMTP URL with a dedicated user account for authentication:  
 ```
-MAILER_URL=smtp://username:password@mx.example.com:587?encryption=tls&auth_mode=plain
+MAILER_URL=smtp://username:password@mx.example.com:587
```
Comment (kevinpapst) [inline _documentation/emails.md:86]:
```suggestion
MAILER_URL=smtps://username:password@mx.example.com:587
```

**Review by kevinpapst — APPROVED** [review]
Thanks 👍

**Pull request merged**